### PR TITLE
Add option sniff (for executing not the whole standard)

### PR DESCRIPTION
**Pull request by jochen42**

### CODE REVIEW

**Comment by JustBlackBird** [comment]
I've reviewed your commit (see https://github.com/jochen42/gulp-phpcs/commit/513ba62228fff2070a1d2607fa4fb169c65f1ff0). Hope you could fix the problems.

File: index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ var gutil = require('gulp-util'),
  * @param {Object} options List of PHP Code Sniffer options.
  * @returns {String} Shell command with all needed flags.
  */
-var buildCommand = function(options) {
+var buildCommand = function (options) {
```
Comment (JustBlackBird) [inline index.js]: Codding style should be untouched.

File: index.js [hunk 2]
```diff
@@ -36,6 +36,10 @@ var buildCommand = function(options) {
         command += ' -s';
     }
 
+    if (opt.hasOwnProperty('sniffs') && typeof opt.sniffs === 'object' && opt.sniffs.length !== 0) {
```
Comment (JustBlackBird) [inline index.js]: Why do you check sniffs for being `object`? It should be an array not an object.

File: index.js [hunk 3]
```diff
@@ -36,6 +36,10 @@ var buildCommand = function(options) {
...
+        command += ' --sniffs=' + opt.sniffs.join(',');
```
Comment (JustBlackBird) [inline index.js]: It would be better to wrapp sniffs list in quote marks (like `--sniffs="first,second,third"`).

File: index.js [hunk 4]
```diff
@@ -43,8 +47,8 @@ var buildCommand = function(options) {
     return command;
 };
 
-var phpcsPlugin = function(options) {
-    return through.obj(function(file, enc, callback) {
+var phpcsPlugin = function (options) {
```
Comment (JustBlackBird) [inline index.js]: Coding style should be untouched.

File: index.js [hunk 5]
```diff
@@ -62,7 +66,7 @@ var phpcsPlugin = function(options) {
         }
 
         // Run code sniffer
-        var phpcs = exec(buildCommand(options), function(error, stdout, stderr) {
+        var phpcs = exec(buildCommand(options), function (error, stdout, stderr) {
```
Comment (JustBlackBird) [inline index.js]: Coding style should be untouched.

File: index.js (hunk 2 above)
Comment (jochen42) [inline index.js]: Hi. you are right, but in javascript you will get 'object' by checking an variable of type array.
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof.
additionaly you i can ad Array.isArray(...)
see here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

File: index.js (hunk 1 above)
Comment (jochen42) [inline index.js]: i am sorry for all that style changes. it was an accident.

File: index.js (hunk 3 above)
Comment (jochen42) [inline index.js]: yes, i can do this. but i don't think that is nessessary, because the class-names there will never contain special characters.

**Comment by jochen42** [comment]
so i just added change according your comments. and hope know it is ok for you. thx, jochen

File: index.js (hunk 2 above)
Comment (JustBlackBird) [inline index.js]: FYI: one can use `a instanceof Array` to check if `a` is an array. Nevertheless `Array.isArray` is fine for node.js.

**Comment by JustBlackBird** [comment]
Thank you! The changes released with `v0.7.0`.

**Pull request merged**